packages/graphql/src/contracts.js

```diff
@@ -91,6 +96,7 @@ function pollForBlocks() {
 }
 
 export function setNetwork(net, customConfig) {
+  if (net !== 'mainnet') console.debug(`Connectiong to network ${net}`)
```

Comment:
Small typo 🙂

Reply:
What?  Do you not like connectiong to things?